core/server/services/routing/helpers/fetch-data.js

```diff
@@ -97,7 +97,7 @@ function fetchData(pathOptions, routerOptions, locals) {
 
     // CASE: fetch more data defined by the router e.g. tags, authors - see TaxonomyRouter
     _.each(routerOptions.data, function (query, name) {
-        props[name] = processQuery(query, pathOptions.slug, locals);
+        props[name] = processQuery(_.merge(query, defaultQueryOptions), pathOptions.slug, locals);
```

Comment:
`defaultQueryOptions` is only valid if the target fetch is from post or page api endpoint.

Reply:
yeah, was thinking about his part as well.... the other way around it would be having a config similar to:
```
const defaultDataQueryOptions = {
    post: {
        options: {
            include: 'author,authors,tags'
        }
    },
    page: {
        options: {
            include: 'author,authors,tags'
        }
    },
    tag: {
        // nothing is needed , same for author
    }
};
```
but with tag and author resources, no additional relations are needed to render collections index page.

Reply:
@kirrg001 maybe adding per resource config like that will make it more explicit.. even if there's nothing to 'extend'. will do the change :+1: